addons/mrp/report/mrp_production_templates.xml

```diff
@@ -76,15 +71,12 @@
                             <tr>
                                 <th><strong>Product</strong></th>
                                 <th class="text-right"><strong>Quantity</strong></th>
-                                <th class="text-center" groups="stock.group_stock_multi_locations">
+                                <th class="hidden text-center" groups="stock.group_stock_multi_locations" >
```

Comment:
I know the spec said *hide* but I think we can just remove these. What's the point of keeping all of these in the report?